<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="grp_cfg">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>100</width>
      <height>350</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QComboBox" name="cmb_port"/>
       </item>
       <item>
        <widget class="QComboBox" name="cmb_baud"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_search">
         <property name="text">
          <string>搜索串口</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_open">
         <property name="text">
          <string>打开</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_close">
         <property name="text">
          <string>关闭</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_clr_rx">
         <property name="text">
          <string>清除接收</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_clr_tx">
         <property name="text">
          <string>清除发送</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ckb_newline">
         <property name="text">
          <string>发送换行符</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ckb_hex_rx">
         <property name="text">
          <string>HEX接收</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ckb_hex_tx">
         <property name="text">
          <string>HEX发送</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="grp_rx">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>280</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPlainTextEdit" name="txt_rx">
       <property name="undoRedoEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="grp_tx">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>500</width>
      <height>70</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPlainTextEdit" name="txt_tx">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>400</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_send">
     <property name="geometry">
      <rect>
       <x>415</x>
       <y>10</y>
       <width>75</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
